### DISCUSSION TITLE
Public Code Announcement - Team ijnek

### DISCUSSION THREAD

**Discussion by ijnek** [cyberbotics/wrestling#103] [General]
As stated in the rules, this is a public code announcement for team ijnek. We will be using ROS 2.
The controller fork can be found at [ijnek_wrestle](https://github.com/ijnek/ijnek_wrestle), but the core logic can be found in a separate repository [wrestle](https://github.com/ijnek/wrestle).
Wish the best for other teams! :+1:

**Comment by omichel** [comment]
Thank you for sharing!
